Answers.md

```diff
@@ -1 +1,13 @@
-
+1. Describe the biggest difference between `.forEach` & `.map`.
+    .for each does not return anything but .map does return
+2. What is the difference between a function and a method?
+    A method is a function but inside of an object
+3. What is closure?
+    Closure is what allows a function to reach outside of themselves and bring in data
+4. Describe the four rules of the 'this' keyword.
+    Implicit
+    Explicit
+    New
+    Window/Global
+5. Why do we need super() in an extended class?
+    It allows us to inherit the properties from the base class
```

Comment:
👍  exactly what I needed to see!

Reply:
Try to work on giving examples as well! This would help you get to understand things a little more.